app/views/login/admin.html.erb

```diff
@@ -0,0 +1,34 @@
" y2="28" stroke="#00E8B0" stroke-linecap="round"></line><path fill="#00E8B0" fill-rule="nonzero" d="M3.53 18.19L0 18.48L0 0.34L9.14 0.34L8.86 3.17L3.53 3.17L3.53 7.87L7.30 7.87L7.30 10.85L3.53 10.85L3.53 18.19ZM15.22 18.65Q12.79 18.65 11.48 16.73Q10.18 14.81 10.18 11.62L10.18 11.62Q10.18 8.45 11.47 6.55Q12.77 4.66 15.22 4.66L15.22 4.66Q17.64 4.66 18.97 6.55Q20.30 8.45 20.30 11.62L20.30 11.62Q20.30 14.78 18.97 16.72Q17.64 18.65 15.22 18.65L15.22 18.65ZM15.22 16.15Q16.30 16.15 16.61 14.83Q16.92 13.51 16.92 11.59L16.92 11.59Q16.92 9.67 16.61 8.35Q16.30 7.03 15.22 7.03L15.22 7.03Q14.14 7.03 13.85 8.28Q13.56 9.53 13.56 11.33L13.56 11.33Q13.56 13.54 13.85 14.84Q14.14 16.15 15.22 16.15L15.22 16.15ZM26.98 18.65Q24.55 18.65 23.24 16.73Q21.94 14.81 21.94 11.62L21.94 11.62Q21.94 8.45 23.23 6.55Q24.53 4.66 26.98 4.66L26.98 4.66Q29.40 4.66 30.73 6.55Q32.06 8.45 32.06 11.62L32.06 11.62Q32.06 14.78 30.73 16.72Q29.40 18.65 26.98 18.65L26.98 18.65ZM26.98 16.15Q28.06 16.15 28.37 14.83Q28.68 13.51 28.68 11.59L28.68 11.59Q28.68 9.67 28.37 8.35Q28.06 7.03 26.98 7.03L26.98 7.03Q25.90 7.03 25.61 8.28Q25.32 9.53 25.32 11.33L25.32 11.33Q25.32 13.54 25.61 14.84Q25.90 16.15 26.98 16.15L26.98 16.15ZM45.31 18.67Q42.22 18.67 40.96 15.86Q39.70 13.06 39.70 8.95L39.70 8.95Q39.70 6.74 40.27 4.70Q40.85 2.66 42.13 1.33Q43.42 0 45.38 0L45.38 0Q46.90 0 48.04 0.94Q49.18 1.87 49.79 3.37Q50.40 4.87 50.40 6.50L50.40 6.50L46.99 6.67Q46.99 5.59 46.63 4.21Q46.27 2.83 45.19 2.83L45.19 2.83Q44.06 2.83 43.63 4.63Q43.20 6.43 43.20 8.95L43.20 8.95Q43.20 12.07 43.68 13.96Q44.16 15.84 45.53 15.84L45.53 15.84Q46.49 15.84 46.90 14.70Q47.30 13.56 47.42 11.93L47.42 11.93L50.42 12.22Q50.42 13.94 49.85 15.42Q49.27 16.90 48.12 17.78Q46.97 18.67 45.31 18.67L45.31 18.67ZM57.12 18.65Q54.70 18.65 53.39 16.73Q52.08 14.81 52.08 11.62L52.08 11.62Q52.08 8.45 53.38 6.55Q54.67 4.66 57.12 4.66L57.12 4.66Q59.54 4.66 60.88 6.55Q62.21 8.45 62.21 11.62L62.21 11.62Q62.21 14.78 60.88 16.72Q59.54 18.65 57.12 18.65L57.12 18.65ZM57.12 16.15Q58.20 16.15 58.51 14.83Q58.82 13.51 58.82 11.59L58.82 11.59Q58.82 9.67 58.51 8.35Q58.20 7.03 57.12 7.03L57.12 7.03Q56.04 7.03 55.75 8.28Q55.46 9.53 55.46 11.33L55.46 11.33Q55.46 13.54 55.75 14.84Q56.04 16.15 57.12 16.15L57.12 16.15ZM64.49 18.48L64.49 5.11L67.85 4.82L67.85 7.01Q68.57 5.98 69.64 5.34Q70.70 4.70 72.36 4.70L72.36 4.70L72.36 7.99Q71.50 7.99 70.09 8.29Q68.69 8.59 67.85 9.34L67.85 9.34L67.85 18.19L64.49 18.48ZM74.18 22.54L74.18 5.11L77.54 4.82L77.54 6.14Q78.34 5.35 79.02 5.00Q79.70 4.66 80.62 4.66L80.62 4.66Q81.86 4.66 82.60 5.35Q83.33 6.05 83.65 7.52Q83.98 9.00 83.98 11.47L83.98 11.47Q83.98 14.86 83.20 16.75Q82.42 18.65 80.21 18.65L80.21 18.65Q79.20 18.65 78.60 18.26Q78 17.88 77.54 17.18L77.54 17.18L77.54 21.86L74.18 22.54ZM79.01 16.08Q79.80 16.08 80.20 15.14Q80.59 14.21 80.59 11.62L80.59 11.62Q80.59 9.14 80.20 8.11Q79.80 7.08 78.98 7.08L78.98 7.08Q78.14 7.08 77.54 8.02L77.54 8.02L77.54 15.00Q78.02 16.08 79.01 16.08L79.01 16.08Z" transform="translate(51, 5.332)"></path></svg>
```

Comment:
This `SVG` here is distracting. What do you think about storing it in its own file (`app/assets/images/logo.svg`, for example), then insert it with `image_tag "logo.svg"`? Same goes for the other svgs in this view. 🤔